.github/workflows/kind.yml

```diff
@@ -488,6 +488,8 @@ jobs:
       run: |
         docker load -i antrea-ubuntu.tar
         docker tag antrea/antrea-ubuntu-coverage:latest antrea/antrea-ubuntu:latest
+        docker tag antrea/antrea-agent-ubuntu-coverage:latest antrea/antrea-agent-ubuntu:latest
```

Comment:
Please check and update `kind-setup.sh`, `test-e2e-kind.sh` and `test-upgrade-antrea.sh` accordingly for these two new images, so we can validate the change via Kind tests.

We can refine Kind related test scripts first in this PR, and submit a separate PR for other CI test scripts to use new images.